apiml-security-common/src/main/java/org/zowe/apiml/security/common/auth/saf/SafResourceAccessEndpoint.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.security.common.auth.saf;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.client.RestTemplate;
+
+@RequiredArgsConstructor
+public class SafResourceAccessEndpoint implements SafResourceAccessVerifying {
+
+    @SuppressWarnings("squid:S1075")
```

Comment:
You don't need to suppress it when you name your constant with `URL` instead of `PATH'.

Reply:
You are right, it is done now